### PR TITLE
Blog post - CloudNativeCon Summary

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
@pilhuhn not sure this is in the right direction... what do you think?

**Comment by jpkrohling** [comment]
@pavolloffay , @objectiser : your feedback is also welcomed!

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+2017-04-07
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, cloudnativecon
+
+CloudNativeCon happened last week in Berlin and Hawkular APM was represented there
+with a talk, link:https://www.youtube.com/watch?v=utCOfSz202o["Operational and Business Insights from Tracing Distributed Microservices"].
+Here's a short summary of the event, with a focus on OpenTracing.
+
+This event was co-located with KubeCon and pretty much everything floated around Kubernetes.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "and as you would expect, pretty much everything revolved around Kubernetes"

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+There were quite a few tracing vendors with booths and/or in the audience:
+Instana, DataDog, Dynatrace, Sysdig, CA Technologies, among others. It was a great
+opportunity to talk to them, demo Hawkular APM to them, watch their demos, exchange ideas
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "demo ...demos" -> "watch each others demos" ?

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+and invite some of them to the OpenTracing community.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "some of" remove? as it sounds like you were being selective :)

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+After the talk, we were able to answer some questions around OpenTracing and
+Hawkular APM, as well as to perform some private demos. Those with good questions got
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: Not sure the ", as well as to perform some private demos" is needed?

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+link:https://pbs.twimg.com/media/C8EgCvzXkAIsr8L.jpg:large["data containers and books"].
+There was also quite some interest in OpenShift: most people knew it already and
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "knew about it"

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+among those who didn't, most seemed impressed by its user interface once presented to it.
+
+On the last day, there was a Distributed Tracing Salon: on this workshop, people
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "in this workshop"

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+were able to get started with OpenTracing by using a demo application written in
+Go and adding instrumentation to it. The final goal was to see the traces on
+Zipkin's user interface, and I believe most participants reached the goal! Those
+who didn't could continue working on it after the workshop, as the
+link:https://github.com/tedsuo/ot-walkthrough-go[source code] is available on GitHub.
+
+It was also a great opportunity to meet some OpenTracing members from LightStep,
+and I made sure to smuggle a few stickers and T-Shirts to Munich, to be distributed
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "back to Munich" ?

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+during meetup talks there. All the videos from this conference, including the
+ones from the OpenTracing track, can be watched on
+link:https://www.youtube.com/playlist?list=PLj6h78yzYM2PAavlbv0iZkod4IVh_iGqV[YouTube].
+
+Some interesting topics were brought up during the conference, a few of them more than once:
+
+* What's the overhead? Hint: OpenTracing is just a specification.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: do we want to say "API" instead or as well as 'specification'?

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc (hunk 3 above)
Comment (jpkrohling) [reply src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: Not being selective, but some of them had components that duplicate things we have in the community, like Java agents. On those cases, I made sure to tell them to join the community, as collaboration would make the agent better for us and for them.

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc (hunk 8 above)
Comment (jpkrohling) [reply src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: "As OpenTracing is an specification and API, the overhead is dependent on the concrete implementation" ?

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc (hunk 3 above)
Comment (objectiser) [reply src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: Understand - the discussions would be specific to the individual cases, but just meant that the blog could be interpreted that way - so better not to include 'some of' to sound more inclusive.

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc (hunk 8 above)
Comment (objectiser) [reply src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: +1 - although 'is a specification' :)

**Comment by pilhuhn** [comment]
I think that looks good (did not "proof-read like @objectiser ). Can you perhaps put the logo of CNCon at the top of the page?

File: src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc (hunk 3 above)
Comment (jpkrohling) [reply src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: How about this? 

> ... invite them to the OpenTracing community. A few of them have components which are similar in purpose to some that we are developing as part of "opentracing-contrib" and would be the perfect opportunity for a collaboration!

Comment (objectiser) [reply src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc]: +1

**Comment by jpkrohling** [comment]
This is ready to be merged on or after Apr 7th.

**Comment by pilhuhn** [comment]
Thanks @jpkrohling

**Pull request merged**